framework/CodeInterfaces/SCALE/tritonAndOrigenData.py

```diff
@@ -0,0 +1,359 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import os
+import copy
+from sklearn import neighbors
+"""
+Created on March 25, 2018
+
+@author: alfoa
+"""
+
+def _scalingFactorBetweenTimeUOM(targetUOM, inputUOM):
+  """
+    Method to get the scaling factor based between UOM for time
+    @ In, targetUOM, str, the target UOM (e.g. s, m,h, d, y)
+    @ In, inputUOM, str, the UOM (e.g. s, m,h, d, y) for which the scaling factor needs to be computed
+    @ Out, multiplier, float, the multiplier to apply
+  """
+  scaling = {'s':1.0,'m':60.0,'h':60.0,'d':24.0,'y':365.25}
+  uoms    = ['s','m','h','d','y']
```

Comment:
would it be better to change "m" into "min"?

Reply:
I use "m" here because ORIGEN uses that for "minutes" and no "months" option is available.